src/dfcx_scrapi/tools/google_drive_connector.py

```diff
@@ -0,0 +1,196 @@
+"""Utility file for Google Sheets"""
```

Comment:
@Hgithubacct I would rename this file to `google_sheets_connector.py` since this Class is exclusively functionality for Google Sheets.